NEWS.md

```diff
@@ -1,5 +1,7 @@
 # pkgdown (development version)
 
+* submenu feature added (@ijlyttle, @wendtke, #1213)
```

Comment:
Added to what? It would also be useful to mention that it only involved new CSS (at least that would've been helpful to me when reviewing)

Reply:
Perhaps the NEWS item could read something like:

```
* add support for navbar submenus: you can create submenus following the convention established in [rstudio/rmarkdown#721](https://github.com/rstudio/rmarkdown/issues/721) (@ijlyttle, @wendtke, #1213)
```

We could have been clearer in the PR conversation that it is only CSS that is changing - I did not think to propose putting it in a NEWS item, as the implementation is not public facing.

We want to get the wording right; hopefully this would get us closer.

Reply:
Thanks for the feedback (@hadley) and suggestions (@ijlyttle). I added the detail in the NEWS bullet and made a commit.